app/src/main/java/org/p2p/wallet/newsend/model/CalculationState.kt

```diff
@@ -0,0 +1,42 @@
+package org.p2p.wallet.newsend.model
+
+import org.p2p.core.utils.Constants.USD_READABLE_SYMBOL
+
+sealed interface CalculationState {
```

Comment:
I don't think State is a good name here
all this classes look like Events: token updated, amount changed, currency switched, etc

Reply:
Sounds good!